package.json

```diff
@@ -51,13 +51,15 @@
     "ember-cli-qunit": "^1.1.0",
     "ember-cli-sri": "^1.2.0",
     "ember-cli-uglify": "^1.0.1",
+    "ember-cli-yuidoc": "offirgolan/ember-cli-yuidoc",
```

Comment:
See cibernox/ember-cli-yuidoc#29

Includes:
- Bug fixes
- Ability to force doc creation in specific env. (this allows easy integration with ember-cli-github-pages) 
- Full yuidoc option support